web/src/api/auth/graphql/update-user.graphql

```diff
@@ -0,0 +1,8 @@
+mutation UpdateUser($id: String!, $_set: user_set_input) {
+  update_user_by_pk(pk_columns: { id: $id }, _set: $_set) {
+    id
+    team_id
```

Comment:
think we don't need this

Reply:
Why? It is still used for the jwt to make requests to Hasura from the user role. For this, we getting team_id from the session.
After this query, we setting the user to the session.